athena/combine_coverage.py

```
@@ -0,0 +1,194 @@
+    # Create output file
+    print(f"Creating combined coverage file: {output_file}")
+    create_combined_xml(combined, output_file)
+    
+    print(f"✅ Combined coverage saved to {output_file}")
+    print(f"📊 Combined {len(combined)} unique packages")
+    
+    # Show summary if requested
+    if show_summary:
+        print("\n📊 Combined Coverage Summary:")
+        print("=" * 50)
+        total_lines = 0
```

Ruff (0.12.2)

Local variable `total_lines` is assigned to but never used

Remove assignment to unused variable `total_lines`

(F841)